widget/embedded/src/hooks/useWalletList.ts

```diff
@@ -175,9 +145,6 @@ export function useWalletList(params: Params) {
     list: sortedWallets.filter(
       (wallet) => !shouldExcludeWallet(wallet.type, chain ?? '', blockchains)
     ),
-    error,
-    handleClick,
-    disconnectConnectingWallets,
-    disconnectWallet,
+    terminateConnectingWallets,
```

Comment:
Considering that you moved all the logic related to handling click, connect and disconnect to `useStatefulConnect` hook, why did you keep `terminateConnectingWallets` here?

Reply:
the logic has broken into two separated part. one is list, and one is for handling to connect to any of them. the list (`useWalletsList`) and connect logic (`useStatefulConnect`). `terminateConnectingWallets` is actually an action the whole list, not related to a single wallet.